symphony/app/fbcnms-projects/magmalte/app/components/KPIGrid.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import DeviceStatusCircle from '@fbcnms/ui/components/icons/DeviceStatusCircle';
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+
+type Status = 'Disabled' | 'Enabled' | 'Down';
+
+type KPIData = {
+  category: string,
+  value: number | string,
+  unit?: string,
+  statusCircle: boolean,
+  status?: Status
+};
+
+export type KPIRows = KPIData[];
+
+type Props = { data: KPIRows[] };
+
+export default function KPIGrid (props: Props) {  
+  const kpiGrid = [];
+  kpiGrid.push(
+    props.data.map((row, i) => (
+      <Grid key={'row-' + i} container direction='row'>
+        {row.map((kpi, j) => (
+          <Grid item xs key={'data-' + i + '-' + j}>
```

Comment:
you can make this easier to read:
```suggestion
          <Grid item xs key={`data-${i}-${j}`}>
```